demos/http/common/src/http_demo_utils.c

```diff
@@ -204,157 +192,3 @@ HTTPStatus_t getUrlAddress( const char * pUrl,
 
     return httpStatus;
 }
-
-/*-----------------------------------------------------------*/
-
-bool getS3ObjectFileSize( size_t * pFileSize,
```

Comment:
It seems that this function is duplicated at more than one place. What is the rationale of moving it out? Why does multithreaded demo needs a separate one?

Reply:
Ignore the multi-threaded part, I see why is it different.